include/wx/generic/grid.h

```diff
@@ -618,7 +618,8 @@ class WXDLLIMPEXP_CORE wxGridHeaderLabelsRenderer
     virtual void DrawHighlighted(const wxGrid& grid,
                                  wxDC& dc,
                                  wxRect& rect,
-                                 int rowOrCol) const;
+                                 int rowOrCol,
+                                 bool pressed = false) const;
```

Comment:
This is [my pet peeve](https://www.wxwidgets.org/develop/coding-guidelines/#no_bool_params) but could we please do something like

```suggestion
                                 int flags = 0) const;
```

and define

```cpp
enum DrawFlags
{
  Draw_Pressed = 1
}
```

here and below?